pkg/metadata/internal/gohai/gohai.go

```diff
@@ -48,7 +48,7 @@ func getGohaiInfo() *gohai {
 		log.Errorf("Failed to retrieve filesystem metadata: %s", err)
 	}
 
-	memoryPayload, err := new(memory.Memory).Collect()
+	memoryPayload, _, err := memory.CollectInfo().AsJSON()
```

Comment:
I kept it the way it was before, but I'm wondering if we should display the list of warnings here in case of error ?
The only possible error which can be returned by `AsJSON` is `no field was collected`